test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -1381,6 +1381,8 @@ BOOST_AUTO_TEST_CASE(internal_constructor)
 			constructor() {}
 		}
 	)";
+	// Disabled via yul because it will throw an error instead of
+	// returning empty bytecode.
```

Comment:
Perhaps have a standard comment structure for explaining like `// viaYul disabled because X`?